src/main/java/com/linecorp/armeria/client/http/ImmutableHttpHeaders.java

```diff
@@ -37,7 +37,7 @@
 
     @Override
     public String get(String name) {
-        return delegate.get(name);
+        return delegate.getAsString(name);
```

Comment:
SuppressWarnings should only apply to this method, not to users of the method (which there can't be since this class is not public).

I think it's intentional for this class to have deprecated methods that call deprecated methods on HttpHeaders since the intention is for it to be exactly the same, except for not exposting mutable methods. But we can clean up the warnings from the armeria build by suppressing here.